RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -2580,9 +2786,17 @@ class DeepTauId : public deep_tau::DeepTauBase {
   const unsigned version_;
   const int debug_level;
   const bool disable_dxy_pca_;
+  const bool disable_hcalFraction_workaround_;
+  const bool disable_CellIndex_workaround_;
   std::unique_ptr<tensorflow::Tensor> tauBlockTensor_;
   std::array<std::unique_ptr<tensorflow::Tensor>, 2> eGammaTensor_, muonTensor_, hadronsTensor_, convTensor_,
       zeroOutputTensor_;
+  CellGrid* inner_grid;
+  CellGrid* outer_grid;
+  const bool save_inputs;
+  std::ofstream* json_file;
+  mutable bool is_first_block;
+  int file_counter;
 
```

Comment:
please stick to the convention of putting underscore at the end of member names

Reply:
Done.